packages/shared/ReactFeatureFlags.js

```diff
@@ -102,6 +102,8 @@ export const enableComponentStackLocations = __EXPERIMENTAL__;
 
 export const enableNewReconciler = false;
 
+export const enableHiddenAPI = false;
```

Comment:
I would call this something that isn't so easily confused with the old API. Even just `enableOffscreenAPI` is slightly better.

Also might as well expose it in the experimental builds. No harm in it.